apps/condo/domains/acquiring/routes/paymentLinkRouter.js

```diff
@@ -180,14 +183,33 @@ class PaymentLinkRouter {
                 )
             } else if (this.isInvoices(req)) {
                 const params = this.extractInvoicesParams(req)
-                await this.redirectIfAlreadyPaid(res, params, this.checkInvoicesAlreadyPaid.bind(this))
```

Comment:
There is was a bug: when there was a redirection in `redirectIfAlreadyPaid`, the code was executed further and resulted in the server crash (the response from the server after we made the redirection)